{"name": "test_login_bemsucedido", "status": "broken", "statusDetails": {"message": "pytest_bdd.exceptions.StepDefinitionNotFoundError: Step definition is not found: When \"eu insiro um email válido \"testeisabelaromanini@teste.com\" e a senha \"Test@2123\"\". Line 4 in scenario \"Login bem-sucedido\" in the feature \"C:\\projetoTCC\\features\\test_login.feature\"", "trace": "E   pytest_bdd.exceptions.StepDefinitionNotFoundError: Step definition is not found: When \"eu insiro um email válido \"testeisabelaromanini@teste.com\" e a senha \"Test@2123\"\". Line 4 in scenario \"Login bem-sucedido\" in the feature \"C:\\projetoTCC\\features\\test_login.feature\"\nAll traceback entries are hidden. Pass `--full-trace` to see hidden and internal frames."}, "description": "../features/test_login.feature: Login bem-sucedido", "start": 1741009181169, "stop": 1741009181169, "uuid": "76385c91-3511-45e9-b9f5-957164ce56ad", "historyId": "f7ac102bbb6279eeeefe88f91dcebd43", "testCaseId": "f7ac102bbb6279eeeefe88f91dcebd43", "fullName": "tests.test_login#test_login_bemsucedido", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "host", "value": "PWIROMANINI"}, {"name": "thread", "value": "44508-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}]}